### PR TITLE
Fix key fuzzer for more coverage

### PR DESCRIPTION
**Pull request by arthurscchan**
#### Summary

Currently, the Keys.constructTufPublicKey method will not be called if Keys.parsePublicKey throws an exception. This PR changes the logic of KeysFuzzer by separating the two method call into two separate fuzzers.

### CODE REVIEW

**Comment by arthurscchan** [comment]
@AdamKorcz

File: fuzzing/src/main/java/fuzzing/KeysFuzzer.java [hunk 1]
```diff
@@ -26,12 +26,20 @@ public static void fuzzerTestOneInput(FuzzedDataProvider data) {
     try {
       String[] schemes = {"rsassa-pss-sha256", "ed25519", "ecdsa-sha2-nistp256"};
       String scheme = data.pickValue(schemes);
+      Boolean choice = data.consumeBoolean();
       byte[] byteArray = data.consumeRemainingAsBytes();
 
-      Keys.parsePublicKey(byteArray);
-      Keys.constructTufPublicKey(byteArray, scheme);
+      if (choice) {
```
Comment (vlsi) [inline fuzzing/src/main/java/fuzzing/KeysFuzzer.java:32]: What do you think of splitting in into several fuzzers?

Comment (arthurscchan) [reply fuzzing/src/main/java/fuzzing/KeysFuzzer.java]: Thanks for the suggestion.

**Comment by loosebazooka** [comment]
replaced by #626

**Pull request closed without merging**